composer.json

```diff
@@ -50,7 +50,8 @@
             },
             "file-mapping": {
                 "[project-root]/.editorconfig": false,
-                "[project-root]/pantheon.upstream.yml": false
+                "[project-root]/pantheon.upstream.yml": false,
```

Comment:
Where is pantheon.upstream.yml coming from?

Reply:
As near as I can tell, pantheon.upstream.yml isn't a scaffolded file. If I'm not mistaken, you could remove this line.